SeaBattle/SeaBattle/Ship.cs

```diff
@@ -28,8 +28,9 @@ public Ship(int length)
 
        public bool addDeck(Content content)
         {
-            if (decks.Count > length)
+            if (decks.Count > length) {

```

Comment:
I think we can leave if body in one line with condition

Reply:
I am very happy that you are able to think but it is important ALWAYS to follow the convention. The convention says that parentheses should be used for if and for and similar blocks. You shouldn't utilize selective approach to following conventions. I am using the same approach in all cases. So, SCREW IT UP.